### PR TITLE
Removing Open CV dependencies

### PR DESCRIPTION
**Pull request by ishaanagw**

### CODE REVIEW

**Comment by ishaanagw** [comment]
Partly fixes - https://github.com/lightly-ai/lightly/issues/1477

File: tests/data/test_VideoDataset.py [hunk 1]
```diff
@@ -51,37 +50,29 @@ def create_dataset_specified_frames_per_video(
...
         self.extensions = ".avi"
 
         for frames in self.frames_over_videos:
             path = os.path.join(self.input_dir, f"output-{len(frames):03}.avi")
             print(path)
-            out = cv2.VideoWriter(path, 0, 1, (w, h))
-            for frame in frames:
-                out.write(frame)
-            out.release()
+            out = torchvision.io.write_video(path, self.frames_over_videos, frames)
```
Comment (guarin) [inline tests/data/test_VideoDataset.py:61]:
```suggestion
            out = torchvision.io.write_video(filename=path, video_array=frames, fps=1)
```

`self.frames_over_videos` is a list of videos and we want to save every video individually.

File: tests/data/test_VideoDataset.py [hunk 2]
```diff
@@ -51,37 +50,29 @@ def create_dataset_specified_frames_per_video(
...
         self.extensions = ".avi"
 
         for i in range(n_videos):
             path = os.path.join(self.input_dir, f"output-{i}.avi")
             print(path)
-            out = cv2.VideoWriter(path, 0, 1, (w, h))
-            for frame in self.frames:
-                out.write(frame)
-            out.release()
+            out = torchvision.io.write_video(path, self.frames, n_frames_per_video)
```
Comment (guarin) [inline tests/data/test_VideoDataset.py:75]:
```suggestion
            out = torchvision.io.write_video(filename=path, video_array=self.frames, fps=1)
```

**Review by guarin — CHANGES_REQUESTED** [review]
Thanks for the PR! I believe the videos are not correctly saved, left some comments :)

**Comment by vectorvp** [comment]
@guarin, is there a way I can help with completing this PR?

**Comment by guarin** [comment]
Hi, yes that would be very helpful. You can merge the changes from this branch into your fork and make a new PR if that is ok?

**Comment by vectorvp** [comment]
@guarin, sure. Are you online during weekends?

**Comment by guarin** [comment]
I'm usually only available during weekdays

**Comment by guarin** [comment]
This has been fixed in #1743

**Pull request closed without merging**